dev/NavigationView/NavigationView.cpp

```diff
@@ -280,7 +280,7 @@ void NavigationView::OnApplyTemplate()
         auto visual = winrt::ElementCompositionPreview::GetElementVisual(topNavOverflowButton);
         CreateAndAttachHeaderAnimation(visual);
 
-#ifdef USE_INSIDER_SDK
+#ifdef USE_INTERNAL_SDK
```

Comment:
These APIs should be in the 18312 SDK, can you please include this under the "USE_INSIDER_SDK" ifdef?